### PR TITLE
[FIX] 2.3.0 hotfix1

### PR DESCRIPTION
**Pull request by luciofm**
Fixes for 2.3.0

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/dagger/qualifier/ForAuthentication.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package chat.rocket.android.dagger.qualifier
+
+import javax.inject.Qualifier
+
+/**
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/dagger/qualifier/ForAuthentication.kt:5]: If you can't now, we'll remove that later 👍

File: app/src/main/java/chat/rocket/android/util/HttpLoggingInterceptor.kt [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+import okhttp3.internal.platform.Platform.INFO
+import okio.Buffer
+import okio.GzipSource
+import java.io.EOFException
+import java.io.IOException
+import java.nio.charset.Charset
+import java.util.concurrent.TimeUnit
+
+/**
+ * An OkHttp interceptor which logs request and response information. Can be applied as an
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/util/HttpLoggingInterceptor.kt:19]: documentation...
💯

File: app/src/main/java/chat/rocket/android/util/extensions/View.kt [hunk 3]
```diff
@@ -5,12 +5,17 @@ import android.support.customtabs.CustomTabsIntent
...
 fun View.openTabbedUrl(url: Uri) {
     with(this) {
         val tabsbuilder = CustomTabsIntent.Builder()
         tabsbuilder.setToolbarColor(ResourcesCompat.getColor(context.resources, R.color.colorPrimary, context.theme))
         val customTabsIntent = tabsbuilder.build()
-        customTabsIntent.launchUrl(context, url)
+        try {
+            customTabsIntent.launchUrl(context, url)
+        } catch (ex: Exception) {
+            Timber.d(ex, "Unable to launch URL")
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/util/extensions/View.kt:18]: 👍

**Review by philipbrito — APPROVED** [review]

File: app/src/main/java/chat/rocket/android/util/HttpLoggingInterceptor.kt (hunk 2 above)
Comment (luciofm) [reply app/src/main/java/chat/rocket/android/util/HttpLoggingInterceptor.kt:19]: 💯 copied from OkHttp logger... It is just a modified/kotlinfied version to remove the auth token from the logs

**Pull request merged**